src/meson.build

```
@@ -0,0 +1,15 @@
+install_headers('ltc.h')
+
+ltc_src = files([
+  'decoder.c',
+  'encoder.c',
+  'ltc.c',
+  'timecode.c'
+])
+
+ltc_lib = library('ltc',
+                  ltc_src,
+                  include_directories : inc,
+                  dependencies : m_dep,
+                  version : '11.1.0',
```

Shouldn't this be `'1.3.1'` or rather:
```suggestion
                  version : meson.project_version(),
```
`meson.`